debug_module.tex

```diff
@@ -23,16 +23,17 @@ \chapter{Debug Module (DM)} \label{dm}
 
 \section{Debug Module Interface (DMI)} \label{dmi}
 
-The Debug Module is a slave to a bus called the Debug Module Interface (DMI). The
+Debug Modules are slaves to a bus called the Debug Module Interface (DMI). The
 master of the bus is the Debug Transport Module(s).
 The Debug Module Interface can be a trivial bus with one master and one slave,
 or use a more full-featured bus like TileLink or the AMBA Advanced Peripheral
 Bus. The details are left to the system designer.
 
 The DMI uses between 7 and 32 address bits.  It supports read and write
 operations.  The bottom of the address space is
-used for the DM. Extra space can be used for custom debug devices, other cores,
-additional DMs, etc.
+used for the first (and usually only) DM. Extra space can be used for custom
+debug devices, other cores, additional DMs, etc. If there are additional DMs
+on this DMI, the DMI address of the next DM is given in \Rnextdm.
```

Comment:
I think you mean the "base address of the next DM on the DMI address space is given in..."

Reply:
Indeed.